reference/js-style.rst

```diff
@@ -10,18 +10,17 @@ First and Foremost
 
 .. Note::
 
-   There are some exceptions for which JSHint complains about things in node
-   that you can ignore, like how it doesn't know what 'const' is and complains
-   about not knowing what 'require' is. You can add keywords to ignore to a
-   `.jshintrc` file.
+   JSHint may warn about some features of Node.js that aren't present in your
```

Comment:
Fwiw we tend to use eslint these days, not JSHint. Probably worth updating this, but I dunno what kind of effort that would be 🤷‍♂️